pallets/stableswap/src/lib.rs

```diff
@@ -565,15 +569,19 @@ pub mod pallet {
 			)
 			.ok_or(ArithmeticError::Overflow)?;
 
+			ensure!(amount >= min_amount_out, Error::<T>::MinimumAmountNotReached);
```

Comment:
Is 0 a valid value for the amount? Can it be 0?

Reply:
you can specify 0 as min amount. up to you.

Reply:
If a user can receive zero amount, then ok.

Reply:
well, then we have some issue somewhere of he specify so very low amount that he does not get anything.

same thing is in omnipool and xyk.